bundles/core/org.openhab.core.library/src/main/java/org/openhab/core/library/types/PointType.java

```diff
@@ -0,0 +1,158 @@
+		this(new DecimalType(latitude.toString()), new DecimalType(longitude.toString()), new DecimalType(altitude.toString()));
+	}
+	
+	public PointType(String value) {
+		if (StringUtils.isNotBlank(value)) {
+			String[] elements = value.split(",");
+			if (elements.length >= 2) {
+				this.latitude = Double.parseDouble(elements[0]);
+				this.longitude = Double.parseDouble(elements[1]);
+				if (elements.length == 3) {
+					this.altitude = Double.parseDouble(elements[2]);
+				}
+			}
+		}
+	}
+		
+	public DecimalType getLatitude() {
+		return new DecimalType(latitude);
+	}
+	
+	public DecimalType getLongitude() {
+		return new DecimalType(longitude);
+	}
+	
+	public DecimalType getAltitude() {
+		return new DecimalType(altitude);
+	}
+	
+	public void setAltitude(DecimalType altitude) {
+		this.altitude = altitude.doubleValue();
+	}
+	
+	public DecimalType getGravity() {
+		double latRad = Math.toRadians(latitude);
+	    double deltaG = -2000.0 * (altitude/1000) * EARTH_GRAVITATIONAL_CONSTANT / ( Math.pow(WGS84_a,3.0) );
+        double sin2lat = Math.sin(latRad) * Math.sin(latRad);
+        double sin22lat = Math.sin(2.0*latRad) * Math.sin(2.0*latRad);
+        double result = (9.780327 * (1.0 + 5.3024e-3 * sin2lat - 5.8e-6 * sin22lat) + deltaG); 
+        return new DecimalType(result);
+	}
+	
+	/**
+	 * <p>Formats the value of this type according to a pattern (@see 
+	 * {@link Formatter}). One single value of this type can be referenced
+	 * by the pattern using an index. The item order is defined by the natural
+	 * (alphabetical) order of their keys.</p>
+	 * 
+	 * @param pattern the pattern to use containing indexes to reference the
+	 * single elements of this type.
+	 */
+	public String format(String pattern) {
+		return String.format(pattern, getConstituents().values().toArray());
+	}
+	
+	public PointType valueOf(String value) {
+		return new PointType(value);
+	}
+		
+	@Override
+	public String toString() {
+		return String.format("%1$.2f°N, %2$.2f°W, %2$.2f m", latitude, longitude, altitude);
+	}
+
+	@Override
+	public SortedMap<String, PrimitiveType> getConstituents() {
+		SortedMap<String, PrimitiveType> result = new TreeMap<String, PrimitiveType>();
+		result.put(KEY_LATITUDE, getLatitude());
+		result.put(KEY_LONGITUDE, getLongitude());
+		result.put(KEY_ALTITUDE, getAltitude());
+		return result;
+	}
+	
+	/**
+	 * Canonicalize the current latitude and longitude values such that:
+	 * 
+	 * <pre>
+	 * -90 &lt;= latitude &lt;= +90 - 180 &lt; longitude &lt;= +180
+	 * </pre>
+	 */
+	private void canonicalize() {
+		latitude = (latitude + 180) % 360;
+		if (latitude < 0) latitude += 360;
+		
+		latitude -= 180;
+		if (latitude > 90) {
+			latitude = 180 - latitude;
+			longitude += 180;
+		} else if (latitude < -90) {
+			latitude = -180 - latitude;
+			longitude += 180;
+		}
+
+		longitude = ((longitude + 180) % 360);
+		if (longitude <= 0) longitude += 360;
+		longitude -= 180;

```

Comment:
it would be really great to have a Unit-Test to proof the correctness of this calculation